### DISCUSSION TITLE
How to configure a Github-App for multiple environments?

### DISCUSSION THREAD

**Discussion by LionelLindt** [community/community#24436] [Apps, API and Webhooks]
<p>Hi there,</p>
<p>I am developing a <a href="https://developer.github.com/apps/building-github-apps/" rel="nofollow noopener">Github-App</a> with rails and I wondered about the configuration of it, when you have multiple domains.</p>
<p>I have a master and development domain on Heroku, like: <em><a href="http://example-develop.herokuapp.com" rel="nofollow noopener">example-develop.herokuapp.com</a></em> and <em><a href="http://example-master.herokuapp.com" rel="nofollow noopener">example-master.herokuapp.com</a></em>.</p>
<p>How would I now configure the app when you can only enter one homepage url? Do I have to setup two apps? One for master and one for develop?</p>
<p>I hope someone can help me with that issue, thank you. <em>Below is an image of the configuration field, that I’m talking about.</em></p>
<p><img src="https://user-images.githubusercontent.com/42026584/181084729-fae4c05d-0158-4d62-97dd-a3955c14afc1.png" alt="github_apps_homepage_url.png" data-base62-sha1="ynYi9jrNxopHxB9StOSQN4TUv1Z" width="483" height="168"></p>

**Comment by lee-dohm** [comment, marked as answer]
<p>Yes, I generally create two (or more) apps, one for development and one for production.</p>
<p>I hope that helps!</p>
